src/bidmanager.js

```diff
@@ -46,33 +46,18 @@ function getBidders(bid) {
 function bidsBackAdUnit(adUnitCode) {
   const requested = $$PREBID_GLOBAL$$._bidsRequested
     .map(request => request.bids
-      .filter(adUnitsFilter.bind(this, $$PREBID_GLOBAL$$._adUnitCodes))
```

Comment:
shouldn't the line below take care of filtering? I don't understand the need for the double filter.